tsl/src/bgw_policy/compression_api.c

```diff
@@ -138,10 +192,38 @@ compress_chunks_add_policy(PG_FUNCTION_ARGS)
 	/* insert a new job into jobs table */
 	namestrcpy(&application_name, "Compress Chunks Background Job");
 	namestrcpy(&compress_chunks_name, "compress_chunks");
-	namestrcpy(&proc_name, "");
-	namestrcpy(&proc_schema, "");
+	namestrcpy(&proc_name, POLICY_COMPRESS_PROC_NAME);
+	namestrcpy(&proc_schema, INTERNAL_SCHEMA_NAME);
 	namestrcpy(&owner, GetUserNameFromId(owner_id, false));
 
+	JsonbParseState *parseState = NULL;
```

Comment:
According to our code style:

```suggestion
	JsonbParseState *parse_state = NULL;
```